src/PluginInstance.ts

```diff
@@ -0,0 +1,15 @@
+import { RispaContext } from './RispaContext'
+
+export default abstract class PluginInstance {
+  context: RispaContext
+
+  constructor(context: RispaContext) {
+    this.context = context
```

Comment:
We need to check what a `context` are passed form super, or do not require override constructor in each plugin. I think first way are more reasonable because we can not garantie what constructor are not overwriten in plugins